Modules/Minimap/MinimapModule.lua

```diff
@@ -19,86 +19,49 @@
 BattlePetCompletionist = LibStub("AceAddon-3.0"):GetAddon("BattlePetCompletionist")
 MinimapModule = BattlePetCompletionist:NewModule("MinimapModule")
 ConfigModule = BattlePetCompletionist:GetModule("ConfigModule")
+BrokerModule = BattlePetCompletionist:GetModule("BrokerModule")
 
-local MinimapIcon = nil
+local LibDataBroker = LibStub("LibDataBroker-1.1", true)
+local LibDBIcon = LibStub("LibDBIcon-1.0", true)
```

Comment:
@GurliGebis Is there a reason that we were loading these modules as optional, despite them being embedded?  If not, we can simplify a bit by removing some conditionals.

Reply:
No, there are no reason, so lets simplify it 😃